conf/configuration.go

```diff
@@ -55,7 +55,11 @@ type JWTConfiguration struct {
 // MFAConfiguration holds all the MFA related Configuration
 type MFAConfiguration struct {
 	ChallengeExpiryDuration float64 `json:"challenge_expiry_duration" default:"300" split_words:"true"`
+	RateLimitChallenge      float64 `split_words:"true" default:"15"`
+	RateLimitVerify         float64 `split_words:"true" default:"10"`
```

Comment:
should we just use 1 config for this? i doubt there's a need to make it so granular

Reply:
sure, let's use just one -- if people ask for it we can reintroduce. Changing now.